spec/requests/storage_services_spec.rb

```diff
@@ -30,4 +30,49 @@
       expect(response.parsed_body).to include(expected)
     end
   end
+
+  context "POST /api/host_initiators" do
+    it "creates new Storage Service" do
+      api_basic_authorize(collection_action_identifier(:storage_services, :create))
+      provider = FactoryBot.create(:ems_autosde, :name => 'Autosde')
+      request = {
+        "action"   => "create",
+        "resource" => {
+          "ems_id"      => provider.id,
+          "name"        => "test_storage_service",
+          "description" => "description of test_storage_service",
+        }
+      }
+      post(api_storage_services_url, :params => request)
+      expect_multiple_action_result(1, :success => true, :message => /Creating Storage Service test_storage_service for Provider #{provider.name}/, :task => true)
+    end
```

Comment:
So the issue we have here is we are depending on the actual class supports values which limits us quite a bit.

The preferred way to do this is to use `stub_supports` to control supports/supports_not without having to change types.

Add the SupportsHelper to this spec:
```
 describe "Storage Services API" do
+  include Spec::Support::SupportsHelper
+
```

Then you can `stub_supports(StorageService, :create)` and `stub_supports(service, :delete)`

Reply:
should I just add the calls to `stub_supports` before the post requests for create/delete? say in lines 46 and 56...

Reply:
Yeah looks good